lib/app/modules/quality_reporting/submit_data_sequence.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require_relative '../../utils/measure_operations'
+require_relative '../../utils/bundle'
+
+module Inferno
+  module Sequence
+    class SubmitDataSequence < SequenceBase
+      include MeasureOperations
+      include BundleParserUtil
+      title 'Submit Data'
+
+      test_id_prefix 'submit_data'
+
```

Comment:
Should have `requires :measure_to_test` to make sure the measure being tested is asked of the user. This sequence attempts to use a blank id otherwise.

Reply:
Good point, although If we do that it'll also add a pop-up to select the measure like in the Measure Availability sequence, which I don't think we want since that allows the user to test submit-data against a measure without first running the Measure Availability sequence for it.

I'm thinking the best approach here would be to put in a null check for `measure_to_test` and if it's null, render an error message instructing the user to first run the Measure Availability sequence.

Reply:
Pushed that change, so lmk if you think that's sufficient.